getting-started/setup-multi-node-auth.md

```diff
@@ -38,26 +131,25 @@ Note that any previously created user passwords will need to be recreated to
 incorporate the new encryption.
 
 ##### Enable password authentication on the data nodes
```

Comment:
Should this really be a separate subsection of step 1?  Might be confusing -> isn't it just part of Step 1?  (Or is it a new step 2?)

Reply:
I've decided to make this a new step 2.